Description: Fix FTCBFS when invoking the external project in GMockConfig.cmake.
Author: Helmut Grohne <helmut@subdivi.de>
Forwarded: https://gitlab.com/ubports/core/cmake-extras/-/merge_requests/4

--- a/src/GMock/GMockConfig.cmake
+++ b/src/GMock/GMockConfig.cmake
@@ -119,7 +119,8 @@
                                            "${findgmock_gmock_main_lib}"
                                            "${findgmock_gmock_lib}"
                           INSTALL_COMMAND ""
-                          CMAKE_ARGS "-DCMAKE_CXX_FLAGS=${findgmock_cxx_flags}")
+                          CMAKE_ARGS "-DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}"
+                                     "-DCMAKE_CXX_FLAGS=${findgmock_cxx_flags}")
 
 add_library(gtest INTERFACE)
 target_include_directories(gtest INTERFACE ${GTEST_INCLUDE_DIRS})
